open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/listeners/AssetLineageOMRSTopicListener.java

```diff
@@ -209,6 +209,10 @@ private void processClassifiedEntityEvent(EntityDetail entityDetail) throws OCFC
         if (!anyLineageClassificationsLeft(entityDetail))
             return;
 
+        if (!publisher.isEntityEligibleForPublishing(entityDetail)) {
```

Comment:
Would it not be better to have a positive if and  put the subsequent 2 lines inthe body of the if. So you do not need to issue the return?

Reply:
I consider an early return makes code more readable and clean.

Reply:
@danielaotelea  For me this is a suggestion, I do not want to battle over style; so for me this can stay as is,  if you do not want to change. I am happy for you to resolve this conversation.